src/frontend/components/Felleskomponenter/Datovelger/Datovelger.test.tsx

```diff
@@ -71,9 +76,12 @@ describe('Datovelger', () => {
             </TestProvidere>
         );
 
-        const tilOgMedÅpneknapp = getAllByRole('button')[1];
+        const [, tilOgMedÅpneknapp] = getAllByRole('button');
```

Comment:
Av nysgjerrighet: Hvordan funker denne måten å skrive det på? `[, tilOgMedÅpneknapp]`

Reply:
Dette er array destructuring, der jeg ser bort fra første (nullte) element. Det er mest digg når du skal lage konstanter ut fra flere elementer i et array, f.eks. `const [idag, imorgen] = listeMedDager`, men jeg synes det også er fint som her når vi vil hoppe over første element. Liker det litt bedre enn å lese indekslesing etter et funksjonskall 😇 

Men det krever at det er lesbart for andre, da. Så om det ikke gir mening trenger vi ikke å gjøre det sånn

Se MDN: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment